routers/api/v1/api.go

```diff
@@ -992,7 +992,7 @@ func Routes() *web.Route {
 				m.Combo("", reqToken()).Get(org.ListTeams).
 					Post(reqOrgOwnership(), bind(api.CreateTeamOption{}), org.CreateTeam)
 				m.Get("/search", org.SearchTeam)
```

Comment:
```suggestion
				m.Get("", org.ListTeams)
				m.Post("", bind(api.CreateTeamOption{}), org.CreateTeam)
				m.Get("/search", org.SearchTeam)
```

Reply:
rm already checked & make it more readable

Reply:
`reqOrgOwnership()` !== `reqOrgMembership()`?